Makefile

```diff
@@ -53,8 +53,12 @@ COMPONENTS              := app_trace \
                            console \
                            cxx \
                            driver \
+			   efuse \
```

Comment:
Please check white space alignment with the rest of the modules in this list.

Reply:
Done, tabs are converted to spaces